server/.env.example

```diff
@@ -0,0 +1,3 @@
+PORT=9000
```

Comment:
why the file is .env.example,. plus env files should not be part of the commit and should be in gitignore

Reply:
I've seen this file been added in few tutorials. Basically .env.example works as a guide to setup the actual .env. It has all the constants variables same as .env but with no actual values but only examples for what kind of value these variables should have. 
env file is already in gitignore.